cdc/owner/ddl_sink.go

```diff
@@ -54,6 +54,7 @@ type DDLSink interface {
 	// the caller of this function can call again and again until a true returned
 	emitDDLEvent(ctx context.Context, ddl *model.DDLEvent) (bool, error)
 	emitSyncPoint(ctx context.Context, checkpointTs uint64) error
+	emitBootstrap(ctx context.Context, bootstrap *model.DDLEvent) error
```

Comment:
Could you add some comments for emitBootStrap? So that we can quickly know the function usage.